core/common/src/rpc/ModelGeometryChanges.ts

```diff
@@ -6,12 +6,8 @@
  * @module RpcInterface
  */
 
-import {
-  assert, CompressedId64Set, DbOpcode, GuidString, Id64String,
-} from "@bentley/bentleyjs-core";
-import {
-  Range3d, Range3dProps,
-} from "@bentley/geometry-core";
+import { assert, CompressedId64Set, DbOpcode, GuidString, Id64String } from "@bentley/bentleyjs-core";

```

Comment:
Did some lint rule require these changes or are they incidental?

Reply:
I think the import sorter vscode plugin did this when I saved the file. Do you think this is too much? I wasn't sure if I should reconfigure it to be more forgiving regarding whitespace.

Reply:
As long as the linter's not going to start hollering about import formatting it's fine with me.